Directory.Build.props

```diff
@@ -5,8 +5,8 @@
   	<PublishWindowsPdb>false</PublishWindowsPdb>
     <ImportNetSdkFromRepoToolset>false</ImportNetSdkFromRepoToolset>
     <Language>C#</Language>
-    <TargetsWindows Condition="'$(RuntimeIdentifier)' == 'win'">true</TargetsWindows>
-    <TargetsLinux Condition="'$(RuntimeIdentifier)' == 'linux'">true</TargetsLinux>
+    <TargetsWindows Condition="$(RuntimeIdentifier.StartsWith('win'))">true</TargetsWindows>
+    <TargetsLinux Condition="$(RuntimeIdentifier.StartsWith('linux'))">true</TargetsLinux>
```

Comment:
Just curious why StartsWith instead of equals?

Reply:
Because the test project's RuntimeIdentifier is not linux, but `linux-arm` instead, so I can't do equals any longer.